src/components/Posts/Posts.jsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from "react";
+import Post from "./Post";
+import { PostContainer } from "./Post.style";
+import { UserName } from "./Posts.style";
+import Logout from "../LogOut/Logout";
+import { PostBtn } from "./Posts.style";
+import NewPost from "./NewPost";
+
+const Posts = (props) => {
+  const [posts, setPosts] = useState([]);
+  const [postStatus, setpostStatus] = useState();
+  const [openPost, setOpenPost] = useState(false);
+  const user = props.user;
```

Comment:
you can save this user in the context then get it from context, for the current project it's not a problem, but in bigger projects you need to logically divide the code as much as possible